dbt-snowflake/src/dbt/adapters/snowflake/impl.py

```diff
@@ -478,3 +478,60 @@ def build_catalog_relation(self, model: RelationConfig) -> Optional[CatalogRelat
             catalog_integration = self.get_catalog_integration(catalog)
             return catalog_integration.build_relation(model)
         return None
+
+    @available
+    def describe_dynamic_table(self, relation: RelationConfig) -> Dict[str, Any]:
+        """
+        Get all relevant metadata about a dynamic table to return as a dict to Agate Table row
+
+        Args:
+            relation (SnowflakeRelation): the relation to describe
+        """
+
+        original_val: Optional[str] = None
```

Comment:
I can probably and should change this to false by default

Reply:
I think that if they haven't set `QUOTED_IDENTIFIERS_IGNORE_CASE` at the session level we shouldn't set it at all and probably still do the unset behavior we have now

Reply:
I checked and I _think_ this is always True or False, but what I made is inherently defensive.